app/templates/components/settings/contact-info-section.hbs

```diff
@@ -0,0 +1,11 @@
+<form class="ui form" action = "">
```

Comment:
For action should be defined like
```hbs
<form class="ui form" {{action 'submit' on='submit'}}>
```

The first `'submit'` refers to the action function name in `app/components/settings/contact-info-section.js` and the second `'submit'` refers to the event that this action should fire upon. Here we want the `submit` action to be called for the `submit` event.